girder/models/model_base.py

```diff
@@ -274,7 +274,12 @@ def remove(self, document, **kwargs):
 
         event = events.trigger('.'.join(('model', self.name, 'remove')),
                                document)
-        if not event.defaultPrevented:
+        kwargsEvent = events.trigger(
+            '.'.join(('model', self.name, 'remove_with_kwargs')), {
+                'document': document,
+                'kwargs': kwargs
+            })
+        if not event.defaultPrevented and not kwargsEvent.defaultPrevented:
```

Comment:
if the model.(model).remove event prevents the default, should the model.(model).remove_with_kwargs event be triggered?

Is there any particular rationale for remove coming before remove_with_kwargs rather than the other way around?